int16 x_go
int16 y_go
float32 r_go
int16 x_ob
int16 y_ob
float32 r_ob
